Source/d_net.c

```diff
@@ -1,580 +1,270 @@
-// Emacs style mode select   -*- C++ -*- 
```

Comment:
Is this the pristine `d_net.c` file from Choco?

Reply:
Yes, but without the support of other games (heretic/hexen/strife)

Reply:
Also, I added sync of Boom/MBF options.

Reply:
I see this is also where `lowres_turn` hides...